pype/plugins/tvpaint/create/create_beauty.py

```diff
@@ -0,0 +1,120 @@
+import avalon.io
+from avalon.tvpaint import pipeline, lib
+
+
+class CreateBeauty(pipeline.Creator):
+    """Mark layer group as one instance."""
+    name = "beauty"
```

Comment:
Please change this from beauty to `RenderLayer`. Beauty is the default pass, but the family is renderLayer.